src/main/java/com/zufar/onlinestore/security/converter/RegistrationDtoConverter.java

```diff
@@ -2,20 +2,10 @@
 
 import com.zufar.onlinestore.security.dto.registration.UserRegistrationRequest;
 import com.zufar.onlinestore.user.dto.UserDto;
-import lombok.RequiredArgsConstructor;
-import org.springframework.stereotype.Service;
+import org.mapstruct.Mapper;
 
-@Service
-@RequiredArgsConstructor
-public class RegistrationDtoConverter {
+@Mapper(componentModel = "spring")
+public interface RegistrationDtoConverter {
 
-    public UserDto toDto(final UserRegistrationRequest userRegistrationRequest) {
-        return UserDto.builder()
-                .firstName(userRegistrationRequest.firstName())
-                .lastName(userRegistrationRequest.lastName())
-                .username(userRegistrationRequest.username())
-                .email(userRegistrationRequest.email())
-                .password(userRegistrationRequest.password())
-                .build();
-    }
-}
+    UserDto toDto(final UserRegistrationRequest userRegistrationRequest);
+}
```

Comment:
Add new empty line at the end of this file

Reply:
done